action.yml

```
@@ -13,341 +13,176 @@
+        source ${{ github.workspace }}/src/package_version_update.sh
+        source ${{ github.workspace }}/src/project.sh
+
+        git_setup_user
+
+        updated_version=$(echo "${{ steps.git_prerequisites.outputs.workspaces_info }}" | awk -F';' '{print $1}' | awk -F':' '{ print $NF }')
+
+        git_create_tag \
+          --version "${updated_version}" \
+          --tag-message "${{ inputs.version_message }} ${updated_version}" ${{ inputs.refresh_minor && '\
+            --refresh-minor' || '' }}
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)